### DISCUSSION TITLE
Any downsides to buying ESP BOX LITE vs ESP BOX?

### DISCUSSION THREAD

**Discussion by tensiondriven** [toverainc/willow#82] [Q&A]
As the title asks, is there any technical downside to going with the ESP BOX LITE vs the ESP BOX?  I see the LITE on Aliexpress for ~35 US vs $55 for the regular BOX.

They seem to have the same chip and RAM, though I saw the build directions are different for the LITE.  Form factor aside, is there any reason not to go with LITE's?

**Comment by mineshaftgap** [comment]
The lite doesn't have the touch screen.

**Reply by kristiankielhofner** [reply]
Also no hardware microphone mute button or stand. Additionally IMO the LCD itself is significantly better on the BOX vs Lite.

In either case Willow + ESP BOX is a tremendous value.
